File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        for(int i = 0; i<nums.size(); i++){
            for(int j = 0; j<nums.size(); j++){
                if(nums[i] == nums[j] && i != j){
                    cout << to_string(nums[i]) + " " + to_string(nums[j]) + "\n";
                    cout << to_string(i) + " " + to_string(j) + "\n";
                    return true;
                }
            }
        }
        return false;
    }
};

File: Data Structures & Algorithms/permutations/submission-0.py
class Solution:
    def permute(self, nums: List[int]) -> List[List[int]]:
        res = []
        def search(n1, n2):
            if len(n2) == 0:
                res.append(n1.copy())
            else:
                for i in range(len(n2)):
                    n1.append(n2.pop(i))
                    search(n1,n2)
                    n2.insert(i, n1.pop())
        search([], nums)
        return res